examples/gocounter/container-deployment/Containerfile.gocounter

```diff
@@ -0,0 +1,25 @@
+FROM golang:1.19 as gocounter-build
+
+RUN apt-get update && apt-get install -y \
+ libcap2-bin \
+ clang \
+ gcc-multilib \
+ libbpf-dev
+
+RUN go install github.com/cilium/ebpf/cmd/bpf2go@master
+
+WORKDIR /usr/src/bpfd/
+COPY ./ /usr/src/bpfd/
+
+WORKDIR /usr/src/bpfd/examples/gocounter
+
+# Compile gocounter
+RUN go generate
+RUN go build
+RUN /sbin/setcap cap_dac_override,cap_bpf=ep ./gocounter
```

Comment:
Is this still supposed to be there?

Reply:
Removed.
As part of this change and retesting, found that I couldn't get it run with a base image of `scratch` or `alpine`. Keep getting the error `exec ./gocounter: no such file or directory`. With one of the runs (can't remember the base image, might have been alpine) I got a little more info:
```
2022/10/07 18:19:22 Program registered with af263886-ce19-4657-8275-d51b3e545fd0 id
2022/10/07 18:19:22 Failed to load pinned Map: /var/run/bpfd/fs/maps/af263886-ce19-4657-8275-d51b3e545fd0/xdp_stats_map
2022/10/07 18:19:22 no such file or directory
```
So it has to do with mounting the map. I was able to get `registry.fedoraproject.org/fedora-minimal:latest` to work, so used that.

Reply:
Hrm that's strange... I'm fine with minimal, in the future we should explore UBI rather than scratch